src/commands/operator/index.ts

```diff
@@ -87,12 +129,15 @@ export default class Operator extends OperatorJobAwareCommand {
       processTransactions: this.processTransactions,
       userWallet,
       lastBlockFilename: 'operator-blocks.json',
+      apiService: this.apiService,
     })
 
     this.jobsFile = path.join(this.config.configDir, this.networkMonitor.environment + '.operator-job-details.json')
 
     // Load the last block height from the file
-    this.networkMonitor.latestBlockHeight = await this.networkMonitor.loadLastBlocks(this.config.configDir)
```

Comment:
I wonder if we should have 1 function that does either. Or maybe an if tree here that chooses how  the block height is run.